crates/telio-nurse/src/qos.rs

```diff
@@ -316,16 +343,16 @@ impl Analytics {
                     .push_str(&node.connected_time.as_secs().to_string());
                 output.connection_duration.push(';');
             } else {
-                output.rtt.push_str(&Analytics::empty_data(buckets));
```

Comment:
Same for this. make a generic method call for all this.

Reply:
I do understand, why you want a one function for 288:296 lines, but this is already quite lean in code ...